framework/atoms/social-icons/social-icons.html

```diff
@@ -0,0 +1,172 @@
+    <li class='icon-social icon-social--delicious'>Delicious</li>
+    <li class='icon-social icon-social--digg'>Digg</li>
+    <li class='icon-social icon-social--diigo'>Diigo</li>
+    <li class='icon-social icon-social--dzone'>Dzone</li>
+    <li class='icon-social icon-social--email'>Email</li>
+    <li class='icon-social icon-social--facebook'>Facebook</li>
+    <li class='icon-social icon-social--flickr'>Flickr</li>
+    <li class='icon-social icon-social--foursquare'>Foursquare</li>
+    <li class='icon-social icon-social--generic'>Generic</li>
+    <li class='icon-social icon-social--google'>Google</li>
+    <li class='icon-social icon-social--gplus'>Gplus</li>
+    <li class='icon-social icon-social--instagram'>Instagram</li>
+    <li class='icon-social icon-social--linkarena'>Linkarena</li>
+    <li class='icon-social icon-social--linkedin'>Linkedin</li>
+    <li class='icon-social icon-social--live'>Live</li>
+    <li class='icon-social icon-social--meneame'>Meneame</li>
+    <li class='icon-social icon-social--misterwong'>Misterwong</li>
+    <li class='icon-social icon-social--myspace'>Myspace</li>
+    <li class='icon-social icon-social--netlog'>Netlog</li>
+    <li class='icon-social icon-social--netvibes'>Netvibes</li>
+    <li class='icon-social icon-social--newsvine'>Newsvine</li>
+    <li class='icon-social icon-social--nujij'>Nujij</li>
+    <li class='icon-social icon-social--pinterest'>Pinterest</li>
+    <li class='icon-social icon-social--pocket'>Pocket</li>
+    <li class='icon-social icon-social--reddit'>Reddit</li>
+    <li class='icon-social icon-social--rss'>Rss</li>
+    <li class='icon-social icon-social--share'>Share</li>
+    <li class='icon-social icon-social--skype'>Skype</li>
+    <li class='icon-social icon-social--slashdot'>Slashdot</li>
+    <li class='icon-social icon-social--sonico'>Sonico</li>
+    <li class='icon-social icon-social--spotify'>Spotify</li>
+    <li class='icon-social icon-social--storify'>Storify</li>
+    <li class='icon-social icon-social--studivz'>Studivz</li>
+    <li class='icon-social icon-social--stumbleupon'>Stumbleupon</li>
+    <li class='icon-social icon-social--technorati'>Technorati</li>
+    <li class='icon-social icon-social--tuenti'>Tuenti</li>
+    <li class='icon-social icon-social--tumblr'>Tumblr</li>
+    <li class='icon-social icon-social--twitter'>Twitter</li>
+    <li class='icon-social icon-social--viadeo'>Viadeo</li>
+    <li class='icon-social icon-social--vine'>Vine</li>
+    <li class='icon-social icon-social--wordpress'>Wordpress</li>
+    <li class='icon-social icon-social--wykop'>Wykop</li>
+    <li class='icon-social icon-social--xerpi'>Xerpi</li>
+    <li class='icon-social icon-social--yammer'>Yammer</li>
+    <li class='icon-social icon-social--youtube'>Youtube</li>
+    <li class='icon-social icon-social--print'>Print</li>
+  </ul>
+</div>
+
+<div class="no-svg" style="float: left; margin-left: 20px;">
```

Comment:
It works as expected. Icons are not provided in DTT's theme: https://github.com/ec-europa/digital-transformation-dev/tree/develop/lib/themes/europa/images/png/icons/social/small